sdk/storage/azure-storage-blobs/test/ut/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ add_executable (
     page_blob_client_test.hpp
     simplified_header_test.cpp
     storage_retry_policy_test.cpp
-    storage_timeout_test.cpp    
```

Comment:
Maybe we should use some format tools for CMake files as well (we already have it for .hpp and .cpp).

[cmake-format](https://cmake-format.readthedocs.io/en/latest/) is a good candidate, I always use it for my own projects.

Reply:
Thank you - I think that would be perfect! I didn't know such tool existed. Let me to make an MQ item for this.

Reply:
#5066